get-gke-credentials/action.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Get GKE Credentials
+description: |-
+  Google Kubernetes Engine (GKE) is a secured and managed Kubernetes service with 
+  four-way auto scaling and multi-cluster support. Use this action to authenticate to
+  a GKE cluster by generating an emphemeral kubeconfig that can be used with kubectl, helm etc.
+author: GoogleCloudPlatform
+
+inputs:
+  credentials:
+    description: |-
+      Optional service account key to use for authentication to GKE. This should be the JSON
+      formatted private key which can be exported from the Cloud Console. The
+      value can be raw or base64-encoded.
+    required: false
+    
+  name:
+    description: |-
+      Name of the cluster to get credentials for.
+    required: true
+
+  location:
+    description: |-
+      Location (Region/Zone) for the cluster.
+    required: true
+
+  project_id:
+    description: |-
+      Project ID where the cluster is deployed.
```

Comment:
Should note where this is set if not set here

Reply:
Done; updated to match other actions